mutagen/oggvorbis.py

```diff
@@ -68,6 +68,8 @@ def __init__(self, fileobj):
                 "page contains a packet too short to be valid")
         (self.channels, self.sample_rate, max_bitrate, nominal_bitrate,
```

Comment:
Looking at the spec https://xiph.org/vorbis/doc/Vorbis_I_spec.html there is a bug here anyway, it should be an unsigned integer.